Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM golang:1.13-alpine as base
```

Comment:
We could use `scratch` as the base image in order to reduce the docker image size.